Node/demo-facebookQuickReplies/app.js

```diff
@@ -0,0 +1,106 @@
+// This loads the environment variables from the .env file
+require('dotenv-extended').load();
+
+var builder = require('botbuilder');
+var restify = require('restify');
+
+var locationPrompt = require('./prompts/location-prompt');
+
+// Utility for simplifying construction of quick replies.
+var quickReplies = require('./facebook/quickreplies');
+
+// Setup Restify Server
+var server = restify.createServer();
+server.listen(process.env.port || process.env.PORT || 3978, function () {
+    console.log('%s listening to %s', server.name, server.url);
+});
+
+// Create chat bot and listen to messages
+var connector = new builder.ChatConnector({
+    appId: process.env.MICROSOFT_APP_ID,
+    appPassword: process.env.MICROSOFT_APP_PASSWORD
+});
+server.post('/api/messages', connector.listen());
+
+var bot = new builder.UniversalBot(connector, [
+    function (session, args, next) {
+        session.beginDialog('/startdemo');
+    },
+    function (session, args, next) {
```

Comment:
The second empty function can be removed. Also, the array enclosure. Just the first function can be passed to the UniversalBot:
````
var bot = new builder.UniversalBot(connector,
  function (session, args, next) {
    session.beginDialog('/startdemo');
  });
````